war/src/main/webapp/scripts/behavior.js

```diff
@@ -161,31 +181,27 @@ Behaviour.start();
    -- Opera 7 fails
 */
 
-function findElementsBySelector(startNode,selector,includeSelf) {
-    if(includeSelf) {
-        function isSelfOrChild(c) {
-          while(true) {
-              if(startNode == c) return true;
-              if(c == null) return false;
-              c = c.parentNode;
-          }
+function findElementsBySelector(startNode, selector, includeSelf) {
+  if (includeSelf) {
+    function isSelfOrChild(c) {
+      while (true) {
+        if (startNode == c) {
+          return true;
+        }
+        if (c == null) {
+          return false;
         }
-        return Array.from(startNode.parentNode.querySelectorAll(selector)).filter(isSelfOrChild);
-    } else {
-        return Array.from(startNode.querySelectorAll(selector));
+        c = c.parentNode;
+      }
     }
+    return Array.from(startNode.parentNode.querySelectorAll(selector)).filter(
+      isSelfOrChild
+    );
+  } else {
+    return Array.from(startNode.querySelectorAll(selector));
+  }
 }
 
-document.getElementsBySelector = function(selector) {
-    return findElementsBySelector(document,selector);
-}
-
-/* That revolting regular expression explained
-/^(\w+)\[(\w+)([=~\|\^\$\*]?)=?"?([^\]"]*)"?\]$/
-  \---/  \---/\-------------/    \-------/
-    |      |         |               |
-    |      |         |           The value
-    |      |    ~,|,^,$,* or =
-    |   Attribute
-   Tag
-*/
```

Comment:
This comment has long been irrelevant, as the regular expression was removed years ago.